examples/vector_databases/cassandra_astradb/Philosophical_Quotes_CQL.ipynb

```diff
@@ -0,0 +1,1353 @@
+{
```

Comment:
<p>Here we could reference and link to https://github.com/openai/openai-cookbook/blob/main/examples/Question_answering_using_embeddings.ipynb as a general guide for this </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openai/openai-cookbook/pull/655/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>